### ISSUE TITLE
Emacs theme for XIKI

### ISSUE THREAD

**Issue by aaron-elkins** [trogdoro/xiki#161]
Hello, 

Can anyone give this emacs theme from the : [Xiki kickstarter site](https://www.kickstarter.com/projects/xiki/xiki-the-command-revolution/description)

I love this theme, it's so simple, clean and beautiful.

See my attachment below.

And thanks.

![screen shot 2016-06-19 at 21 19 14](https://cloud.githubusercontent.com/assets/5492042/16177551/d68a492e-3663-11e6-8f20-0cf614add95c.png)

**Comment by aaron-elkins** [comment]
I customized my emacs, so close to this theme except the background color.

![screen shot 2016-06-20 at 22 19 12]
(https://cloud.githubusercontent.com/assets/5492042/16197544/0bb7dbe4-3735-11e6-9a68-ae9992160698.png)

**Comment by trogdoro** [comment]
Heh, yeah that looks about right! I was going to suggest digging through
the Xiki source on github but it looks like you've got it.

--Craig

**Comment by aaron-elkins** [comment]
Yep, all is right except that i can not customize height for the bottom mini buffer (or echo area), any suggestions?

After looking into the Xiki source, i found nothing about this issue.
